infra/alloy/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM grafana/alloy:latest
```

Comment:
**suggestion (non-blocking):** Please lock the image, including the patch version.